src/main.rs

```diff
@@ -154,19 +170,7 @@ fn command_new_peer(cfg: &mut configs::WireguardNetworkInfo, matches: &clap::Arg
         ips: vec![],
     };
 
-    if let Some(template_name) = matches.value_of("use-template") {
```

Comment:
Can we instead add separate "validate_config", so we don't start passing everything into edit command?

Reply:
`Validate_config` sounds like validating after modifying, and it is not clear what to validate, everything? 

We need to only check that the specified peer exists. I like how it is checked exactly in the place where it is used, like in `command_export` or `command_remove`.

What are other alternatives to check peer existence?

Reply:
Well, for instance https://github.com/cab404/wg-bond/pull/18/commits/3ee221027dd7eb492cc64cecf7b234cecf1c30c9

Reply:
And also we still need to validate whole config. E.g removing a template which has children could leave config in an incorrect state.

Reply:
Okay. Can I wait until https://github.com/cab404/wg-bond/pull/18 is merged so I can use that validation?

In config validation I can think of:
- All used templates are present
- All used gateways are present
- Templates don't form cycles

Is that what we expect from the validation currently?

Reply:
Sure)